pkg/tools/etcd_tools.go

```diff
@@ -418,20 +418,24 @@ func (h *EtcdHelper) AtomicUpdate(key string, ptrToType runtime.Object, ignoreNo
 	}
 }
 
-func checkEtcd(host string) error {
+// Performs a version check against the provided Etcd server, returning a triplet
+// of the release version, internal version, and error (if any).
+func CheckEtcd(host string) (string, string, error) {
```

Comment:
Consider naming the return parameters? Also should this function now be called GetEtcdVersions or something?